install_config/topics/configuring_for_vsphere_registry.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * install_config/configuring_for_vsphere.adoc
+
+[id='vsphere-configuring-registry_{context}']
+= Configuring the {product-title} registry for vSphere
+
+
+[source, bash]
+----
+VMFS
+cd /vmfs/volumes/datastore1/
+# VSAN
+cd /vmfs/volumes/vsanDatastore/
+
+# VMFS
+mkdir kubevols # Not needed but good hygiene
+# VSAN
+/usr/lib/vmware/osfs/bin/osfs-mkdir kubevols # Needed
+
+cd kubevols
+
+vmkfstools -c 25G registry.vmdk
+----
+
+== Configuring the {product-title} registry for vSphere using Ansible
+
+.Procedure
+
+To configure the Ansible inventory for the registry to use a vSphere volume:
+
+----
+[OSEv3:vars]
+# vSphere Provider Configuration
+openshift_hosted_registry_storage_kind=vsphere <1>
+openshift_hosted_registry_storage_access_modes=['ReadWriteOnce'] <2>
+openshift_hosted_registry_storage_annotations=['volume.beta.kubernetes.io/storage-provisioner: kubernetes.io/vsphere-volume'] <3>
```

Comment:
Do the square brackets need to be here? As with the line above? There's another part of the doc where they are needed, so it's worth asking. Alternatively, they might be signifying replaceable values here...

Reply:
They are required. The variables in question are loading into list values.